pyanaconda/addons.py

```diff
@@ -103,6 +107,7 @@ def setup(self, storage, ksdata, payload):
         self.__dict__ = d
         for v in self.__dict__.values():
             if hasattr(v, "setup"):
+                log.debug("Loading %s addon", v.name)
```

Comment:
Please, change it to `progress_message(N_("Setting up %s addon") % v.name)`

Reply:
Fixing now.